doc/wiki/dashboard-audit.md

```diff
@@ -0,0 +1,668 @@
+## Dashboard query audit
+Reviewing the following Dashboard queries to ensure we're displaying and describing the data correctly:
+
+1. New registered patients
+2. Total registered patients
+3. Total assigned patients
+4. BP controlled
+5. BP not controlled
+6. Visited but no BP taken
+7. Missed visits
+8. Lost to follow-up
+9. Follow-up patients
+10. Inactive facilities
+11. BP measure taken
+12. BP log
+13. Cohort reports
+
+---
+
+## Monthly registered patients
+**Copy used in the Dashboard**
+Hypertension patients registered during a month in `region_name`
+
+**How it's calculated**
+The number of patients registered at a facility during a month where the patient:
++ Is not deleted
++ The patient is hypertensive
+
+*Sample SQL query*
+```sql
+SELECT Count(DISTINCT "patients"."id") AS count_id,
+       Date_trunc('month', "patients"."recorded_at"::timestamptz AT TIME ZONE 'ETC/UTC') AT TIME ZONE 'ETC/UTC' AS date_trunc_month_patients_recorded_at_timestamptz_at_time_zone_
+FROM "patients"
+INNER JOIN "medical_histories" ON "medical_histories"."deleted_at" IS NULL
+AND "medical_histories"."patient_id" = "patients"."id"
+WHERE "patients"."deleted_at" IS NULL
+  AND "patients"."registration_facility_id" = $1
+  AND "medical_histories"."deleted_at" IS NULL
+  AND "medical_histories"."hypertension" = $2
+  AND ("patients"."recorded_at" IS NOT NULL)
+GROUP BY date_trunc('month', "patients"."recorded_at"::timestamptz AT TIME ZONE 'ETC/UTC') AT TIME ZONE 'ETC/UTC' [["registration_facility_id", "2e768fb2-9de0-4a7d-a64d-9f5b4c1863f4"],["hypertension", "yes"]]"
+```
+
+**Where it's shown in the Dashboard** 
++ **Reports:** Overview, Details
++ **Home:** BP controlled, BP not controlled, Missed visits
++ **Simple app:** Progress tab
+
+**Things we should fix**
++ [ ] Rename query name from "Monthly registered patients" to "New registered patients"
```

Comment:
Your suggestion is clearer, @kpethtel. But I'm suggesting we remove "Monthly" because we only show month-wise data in the Dashboard and feels a bit redundant. We'll see what the rest of the team says.